pkg/workflow/workflowrun/workload.go

```diff
@@ -75,8 +78,21 @@ func (p *WorkloadProcessor) processPod() error {
 	}
 	log.WithField("stg", p.stg.Name).Debug("Pod manifest created")
 
-	// Create the generated pod.
-	po, err = p.clusterClient.CoreV1().Pods(pod.GetExecutionContext(p.wfr).Namespace).Create(po)
+	// Create the generated pod with retry on exceeded quota.
+	// Here is a litter tricky. Cyclone will delete stage related pod to release cpu/memory resource when stage have
+	// been finished, but pod deletion needs some time, so retry on exceeded quota gives the time to waiting previous
+	// stage pod deletion.
+	backoff := wait.Backoff{
+		Steps:    controller.Config.RetrySettings.Count,
+		Duration: time.Duration(controller.Config.RetrySettings.Period) * time.Second,
+		Factor:   1.0,
```

Comment:
If the resources are not sufficient, Cyclone should wait more time before next retry. I think this should be added into retry config, and set a reasonable default value.